src/App.xaml.cs

```diff
@@ -135,7 +138,11 @@ public App()
 
             // Main window: Allow access to the main window
             // from anywhere in the application.
-            services.AddSingleton<WindowEx>(_ => MainWindow);
+            services.AddSingleton(_ => MainWindow);

```

Comment:
I have [an issue](https://github.com/microsoft/devhome/issues/1731) to do exactly this, but I do have some enhancements we can do as well 😊

Because the Window and the DispatcherQueue are sometime referenced/injected in testable classes, we should consider mapping an interface to them.

Reply:
Something like this:
```
AddSingleton<IAppWindow>(_ => MainWindow);
AddSingleton<IDispatcherQueue>(_ => MainWindow.DispatcherQueue);
```

Another option is to add a reference to the `MainWindow` and `DispatcherQueue` in `IApp`